### PR TITLE
Changed page to account for client server split. Also included links …

### PR DESCRIPTION
**Pull request by code-11**
…to all resources of the project.

### CODE REVIEW

File: projects/food-truck-locator.md [hunk 1]
```diff
@@ -5,12 +5,21 @@ Food Truck Locator
...
-* [Posted on Pivotal Tracker](https://www.pivotaltracker.com/n/projects/1430534)
\ No newline at end of file
+
+### Pieces
+
+* [Server](https://github.com/code-11/FoodTruckLocatorServer)
+* [Client](https://github.com/code-11/FoodTruckLocatorClient)
+* [Pivotal Tracker](https://www.pivotaltracker.com/n/projects/1430534)
+* [Heroku Client](https://github.com/code-11/FoodTruckLocatorClient)
+* [Heroku Server](https://github.com/code-11/FoodTruckLocatorClient)
```
Comment (dblock) [inline projects/food-truck-locator.md]: This looks incorrect, it should be `Server`, no?

Comment (dblock) [inline projects/food-truck-locator.md]: Actually both are incorrect, those shouldn't be github links but heroku.

Comment (code-11) [inline projects/food-truck-locator.md]: Good catch, will change.

**Comment by code-11** [comment]
Closing pull request since I royally messed up rebase and had to reclone the repo. New, clean and better pull request incoming.

**Comment by dblock** [comment]
It's a good place to learn how to use `git commit --amend` ;)

**Pull request closed without merging**